tests/test_logging.py

```diff
@@ -178,59 +178,104 @@ def setup_logging_config(cli_options: Options, file_option_value: Optional[str]
+    options1 = {"log_file": old_log_file, "log_file_level": "4", "keep_logger_names": False}
+    setup_logging_config(
+        cli_options=Options(logging_config=path_logging_config_file1, **options1),
+        file_option_value=None,
+    )
     logger.info("test")
-    assert "AAA test" in stream.getvalue()
+    output = get_output_and_clear()
+    assert "AAA test" in output
+    assert f"{options1} options were ignored. Using logging config from file {path_logging_config_file1}" in output
 
     # Set logging_config option in cfg file only
     write_logging_config_file(path=path_logging_config_file1, formatter="BBB %(message)s")
-    setup_logging_config(cli_options=Options(), file_option_value=path_logging_config_file1)
+    options2 = {"keep_logger_names": False}
+    setup_logging_config(
+        cli_options=Options(**options2),
+        file_option_value=path_logging_config_file1,
+    )
     logger.info("test")
-    assert "BBB test" in stream.getvalue()
+    output = get_output_and_clear()
+    assert "BBB test" in output
+    assert f"{options2} options were ignored. Using logging config from file {path_logging_config_file1}" in output
 
     # Set the logging-config config option both on CLI and cfg file. CLI option takes precedence.
     write_logging_config_file(path=path_logging_config_file1, formatter="CCC %(message)s")
     write_logging_config_file(path=path_logging_config_file2, formatter="DDD %(message)s")
     setup_logging_config(
-        cli_options=Options(logging_config=path_logging_config_file1),
+        cli_options=Options(logging_config=path_logging_config_file1, **options2),
         file_option_value=path_logging_config_file2,
     )
     logger.info("test")
-    assert "CCC test" in stream.getvalue()
+    output = get_output_and_clear()
+    assert "CCC test" in output
+    assert f"{options2} options were ignored. Using logging config from file {path_logging_config_file1}" in output
 
     # Set the logging-config config option in the cfg config file and using the environment variable.
     # The environment variable takes precedence.
     write_logging_config_file(path=path_logging_config_file1, formatter="EEE %(message)s")
     write_logging_config_file(path=path_logging_config_file2, formatter="FFF %(message)s")
+    options3 = {"timed": True, "keep_logger_names": False}
     with monkeypatch.context() as m:
         m.setenv("INMANTA_CONFIG_LOGGING_CONFIG", path_logging_config_file1)
         setup_logging_config(
-            cli_options=Options(),
+            cli_options=Options(**options3),
             file_option_value=path_logging_config_file2,
         )
         logger.info("test")
-        assert "EEE test" in stream.getvalue()
+        output = get_output_and_clear()
+        assert "EEE test" in output
+        assert f"{options3} options were ignored. Using logging config from file {path_logging_config_file1}" in output
```

Comment:
This message is not in line with the comment above saying "The environment variable takes precedence".

Reply:
What do you mean? Don't we set the env variable above with:
`m.setenv("INMANTA_CONFIG_LOGGING_CONFIG", path_logging_config_file1)`